Nav.php

```diff
@@ -202,6 +202,32 @@ public function renderItem($item)
     }
 
     /**
+     * Check if parent item is visible
+     * @param array $parentItem
+     * @return boolean
+     */
+    protected function isParentVisible($parentItem)
```

Comment:
I think it's more appropriate because only parent items (of the first level) checked like that, and item can be considered as parent even it don't have any children.